sdk/rust/oak_tests/src/lib.rs

```diff
@@ -33,24 +33,38 @@ use tonic::{
     Request,
 };
 
+pub enum Profile {
+    Release,
+    Debug,
+}
+
 // TODO(#544): re-enable unit tests of SDK functionality
 
 /// Uses cargo to compile a Rust manifest to Wasm bytes.
-pub fn compile_rust_wasm(cargo_path: &str, module_wasm_file_name: &str) -> anyhow::Result<Vec<u8>> {
+pub fn compile_rust_wasm(
+    cargo_path: &str,
+    module_wasm_file_name: &str,
+    profile: Profile,
+) -> anyhow::Result<Vec<u8>> {
     // Use a separate target dir for Wasm build artifacts. The precise name is not relevant, but it
     // should end with `target` so that it gets automatically ignored by our `.gitignore`.
     let target_dir = PathBuf::from("oak_tests/target");
 
+    let mut args = vec![
+        "build".to_string(),
+        format!(
+            "--target-dir={}",
+            target_dir.to_str().expect("invalid target dir")
+        ),
+        "--target=wasm32-unknown-unknown".to_string(),
+        format!("--manifest-path={}", cargo_path),
+    ];
+    if let Profile::Release = profile {
+        args.push("--release".to_string());
+    }
```

Comment:
This should probably be a `match`, so that if we add more values to the enum, the compiler will warn us that we need to handle all cases here too.

Reply:
Done